### PR TITLE
Allow user to set max results

### PR DESCRIPTION
**Pull request by mrcaron**
Hi! I found that I only wanted one result, so I updated your module to let me set the number of results rather than max out a 5 all the time.

### CODE REVIEW

**Comment by mrcaron** [comment]
btw... don't forget to ignore my travis build badge link when you merge in. :)

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Variable | Default | Description
 `GOOGLE_API_KEY` | N/A | A unique developer [API key](https://developers.google.com/custom-search/json-api/v1/introduction#identify_your_application_to_google_with_api_key) is required to use Google's Custom Search API
 `GOOGLE_CUSTOM_SEARCH` | N/A | The [Google Custom Search][googlesearch] engine [identifier](https://cse.google.com/cse/all) (the `cx` portion of the custom search engine URL)
 `BING_SEARCH_API_KEY` | N/A | The [API key][cognitivebingkey] for performing [Bing Search API][cognitivebing] queries
+`MAX_SEARCH_RESULTS` | 5 | The number of search results to show at maximum
```
Comment (MrSaints) [inline README.md:47]: I think we could simplify the phrasing here. Possibly _"The number of search results to return"_? What are your thoughts? It just seems slightly redundant to say "maximum".

File: src/search.coffee [hunk 2]
```diff
@@ -24,6 +24,10 @@ GOOGLE_API_KEY = process.env.GOOGLE_API_KEY
 GOOGLE_CUSTOM_SEARCH = process.env.GOOGLE_CUSTOM_SEARCH
 BING_SEARCH_API_KEY = process.env.BING_SEARCH_API_KEY
 USE_BING_V5 = process.env.USE_BING_V5
+MAX_RESULTS = process.env.MAX_SEARCH_RESULTS
```
Comment (MrSaints) [inline src/search.coffee:27]: Could we not do `MAX_RESULTS = process.env.MAX_SEARCH_RESULTS or 5` ?

**Review by MrSaints — CHANGES_REQUESTED** [review]
Looks good overall. However, I'd strongly suggest creating a new branch with only the essential commits (i.e. ignoring the changes to the Travis badge for example).

File: README.md (hunk 1 above)
Comment (mrcaron) [reply README.md:47]: yeah, it is. I was just trying to trigger a push to get Travis to build it :)

File: src/search.coffee (hunk 2 above)
Comment (mrcaron) [reply src/search.coffee:27]: Yep! I'm not a js/coffee ninja. I'll change that.

**Pull request closed without merging**